src/components/10-atoms/textarea/index.js

```diff
@@ -256,7 +271,7 @@ class AXATextarea extends NoShadowDOM {
                 <span>${modelCounter}</span>
```

Comment:
you could use if/else here, because when for example `this.showCounter` is true than `this.showCounterMax` and `this.showError` are never going to be true

Reply:
`this.showError` have higher priority then the other two function, because of readability of markup i don't change